etc/format_code.jl

```diff
@@ -0,0 +1,46 @@
+using Pkg
+Pkg.activate(; temp=true)
+Pkg.add("Test")
+Pkg.add(name="JuliaFormatter", version="2")
```

Comment:
```suggestion
Pkg.add(name="JuliaFormatter", version="1")
```
The switch from v1 to v2 needs a reformatting of all files. Thus, I would prefer it not to do it in this PR.
Furthermore, it produces a lot of parsing errors like
```julia
┌ Warning: Failed to format file /home/runner/work/Oscar.jl/Oscar.jl/experimental/BasisLieHighestWeight/src/LieAlgebras.jl due to a parsing error, skipping file
│   error =
│    ParseError:
│    # Error @ line 21:7
│        # operators act downwards, so we need to swap e and f
│    #     ┌
│        if
│        ((fl, i)=is_positive_root_with_index(op); fl)
│    #──┘ ── missing condition in `if`
└ @ JuliaFormatter ~/.julia/packages/JuliaFormatter/tKKVP/src/JuliaFormatter.jl:448
```